### DISCUSSION TITLE
How to deploy and maintain thousands of machines to participate in the Chia project? pls help me .

### DISCUSSION THREAD

**Discussion by ben00007** [Chia-Network/chia-blockchain#5272] [Support]
Because I have thousands of machines, I encounter many confused problems. For example, when 2000 harvesters are connect  a farmer, the CPU utilization rate of the farmer process will reach 100%, so I especially hope to get some help. 1. How to deploy and maintain thousands of machines to participate in the Chia project? 2. Can a fullnode have multiple farmers? 3. Can farmer and fullnode be deployed on different machines? 4. how many investors can a farmer connect to?

**Comment by BrandtH22** [comment]
If you are still experiencing the issues it is recommended to upgrade to the latest version of chia. Please download it from here: https://www.chia.net/downloads/

Since this issue has been open for a number of years without additional comments, we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions, comments, or if the issue persists after an update.

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
